samples/ImplicitFlow/AureliaApp/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "aurelia-app",
+  "description": "An Aurelia client application.",
+  "version": "0.1.0",
+  "repository": {
+    "type": "???",
```

Comment:
@PinpointTownes Done. For now, I used `https://github.com/openiddict/openiddict-samples.git` as the location.